ironic-deployment/keepalived-tls/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: metal3
+resources:
+- ../keepalived
+
+configMapGenerator:
+- behavior: merge
+  literals:
+  - CACERT_FILE=/tls/ca/ca.crt
```

Comment:
Do you mean to use client certificate validation? As I mentioned on the ironic-image review, I'd skip it, at least for now. We don't have a way to provide client keys to the ramdisk (nor do you propose one here).

Reply:
no, the mTLS part is not supported due to issue with PHA in go. We are going for plain TLS, but we want to be able to accept any certificate genearted by cert-manager using that CA

Reply:
I'll remove the CA part from Ironic configuration. it is only needed in BMO actually, considering Ironic has the certificates.

Reply:
@maelk "the mTLS part is not supported due to issue with PHA in go"
Can you please be more specific about "PHA in go"?
What this issues are?

Reply:
This is the exact issue :https://github.com/golang/go/issues/40521 Go does not support Post Handshake Authentication unfortunately, and our solution design was relying on PHA to separate the IPA endpoints that should not be authenticated from the rest : https://github.com/metal3-io/metal3-docs/blob/master/design/ironic_authentication.md

Reply:
@maelk How about just using different DNS records that resolve to same IP?

Reply:
We are not using dns, but directly IP addresses. We cannot easily use different endpoints, so we are waiting for the ironic mTLS feature on IPA, that is WIP in Ironic.